### PR TITLE
Created a new file

### PR DESCRIPTION
**Pull request by Uwampojejecharly**

### CODE REVIEW

File: CharlyUwampojejeCV.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+body{
+    background-image: linear-gradient(#636162, #b1afaf);
+    height: 122vh;
+    letter-spacing: 1px;
+}
```
Comment (fayabobo) [inline CharlyUwampojejeCV.css:6]: We need a line between the 2 css rules.

File: CharlyUwampojejeCV.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+*{
+    margin: 0px; padding: 0px; font-family: 'montserrat', century gothic;
+}
```
Comment (fayabobo) [inline CharlyUwampojejeCV.css:14]: We need an empty line.

File: CharlyUwampojejeCV.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+.main{
+    background-color: white;
+    height: auto;
+    width: 700px;
+    position: absolute;
+    transform: translate( -50%, -50%);
+    left: 50%;
+    top: 60%;
+}
+.left{
```
Comment (fayabobo) [inline CharlyUwampojejeCV.css]: We need an empty line

File: CharlyUwampojejeCV.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    padding: 10px;
+}
+
+.profile-img{
+    height: 180px;
+    width: 180px;
+    border-radius: 60%;
+    margin-bottom: 30px;
+    margin-left: 10px;
+}
```
Comment (fayabobo) [inline CharlyUwampojejeCV.css:44]: We need an empty line

File: CharlyUwampojejeCV.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+.profile-img img{
+    height: 200px;
+    width: 200px;
+    border-radius: 100px;
+
+}
```
Comment (fayabobo) [inline CharlyUwampojejeCV.css:51]: We need an empty line

File: CharlyUwampojejeCV.css [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    font-size: 70%;
+    padding: 0px 30px;
+}
+
+.name-div{
+    padding:70px 0px 60px 55px;
+    text-align: start;
+    letter-spacing: 2px;
+    font-size: large;
+}
```
Comment (fayabobo) [inline CharlyUwampojejeCV.css:63]: We need an empty line

File: CharlyUwampojejeCV.css [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+.nom{
+    font-size: 250%;
+}
+.prenom{
```
Comment (fayabobo) [inline CharlyUwampojejeCV.css]: We need an empty line

Comment (fayabobo) [inline CharlyUwampojejeCV.css]: Change the class name. It has to be in english.

File: CharlyUwampojejeCV.css [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    padding: 0px 30px;
+}
+
+.name-div{
+    padding:70px 0px 60px 55px;
+    text-align: start;
+    letter-spacing: 2px;
+    font-size: large;
+}
+.nom{
```
Comment (fayabobo) [inline CharlyUwampojejeCV.css]: Change the class name. It has to be in english.

**Review by fayabobo — CHANGES_REQUESTED** [review]
You have a few things to correct.

File: CharlyUwampojejeCV.css (hunk 1 above)
Comment (Uwampojejecharly) [reply CharlyUwampojejeCV.css:6]: Done

File: CharlyUwampojejeCV.css (hunk 3 above)
Comment (Uwampojejecharly) [reply CharlyUwampojejeCV.css]: Done

File: CharlyUwampojejeCV.css (hunk 5 above)
Comment (Uwampojejecharly) [reply CharlyUwampojejeCV.css:51]: Done

File: CharlyUwampojejeCV.css (hunk 7 above)
Comment (Uwampojejecharly) [reply CharlyUwampojejeCV.css]: Done

File: CharlyUwampojejeCV.css (hunk 8 above)
Comment (Uwampojejecharly) [reply CharlyUwampojejeCV.css]: Done

File: CharlyUwampojejeCV.css (hunk 2 above)
Comment (Uwampojejecharly) [reply CharlyUwampojejeCV.css:14]: Done

**Pull request merged**